Dockerfile

```diff
@@ -1,161 +1,101 @@
-    #
-    
-RUN mkdir /build && \
-    cd /build && \
-    wget --no-verbose https://github.com/greenbone/openvas-scanner/archive/$openvas_scanner_version.tar.gz && \
-    tar -zxf $openvas_scanner_version.tar.gz && \
-    cd /build/*/ && \
-    mkdir build && \
-    cd build && \
-    cmake -DCMAKE_BUILD_TYPE=Release .. && \
-    make && \
-    make install && \
-    cd / && \
-    rm -rf /build
-    
-    #
-    # Install Greenbone Security Assistant (GSA)
-    #
-    
-RUN mkdir /build && \
-    cd /build && \
-    wget --no-verbose https://github.com/greenbone/gsa/archive/$gsa_version.tar.gz && \
-    tar -zxf $gsa_version.tar.gz && \
-    # https://github.com/Secure-Compliance-Solutions-LLC/GVM-Docker/issues/115
-    sed -i 's/300000/90000000/g' /build/*/gsa/src/gmp/gmpsettings.js && \
-    find /build/ -type f -exec sed -i 's/timeout: 30000/timeout: 9000000/g' {} \; && \
-    find /build/ -type f -exec sed -i 's/expect(settings.timeout).toEqual(30000)/expect(settings.timeout).toEqual(9000000)/g' {} \; && \
-    cd /build/*/ && \
-    mkdir build && \
-    cd build && \
-    cmake -DCMAKE_BUILD_TYPE=Release .. && \
-    make && \
-    make install && \
-    cd / && \
-    rm -rf /build
-    
-    #
-    # Install Greenbone Vulnerability Management Python Library
-    #
-    
-RUN pip3 install python-gvm==$python_gvm_version
-    
-    #
-    # Install Open Scanner Protocol daemon (OSPd)
-    #
-    
-RUN mkdir /build && \
-    cd /build && \
-    wget --no-verbose https://github.com/greenbone/ospd/archive/$open_scanner_protocol_daemon.tar.gz && \
-    tar -zxf $open_scanner_protocol_daemon.tar.gz && \
-    cd /build/*/ && \
-    python3 setup.py install && \
-    cd / && \
-    rm -rf /build
-    
-    #
-    # Install Open Scanner Protocol for OpenVAS
-    #
-    
-RUN mkdir /build && \
-    cd /build && \
-    wget --no-verbose https://github.com/greenbone/ospd-openvas/archive/$ospd_openvas.tar.gz && \
-    tar -zxf $ospd_openvas.tar.gz && \
-    cd /build/*/ && \
-    python3 setup.py install && \
-    cd / && \
-    rm -rf /build
-    
-    #
-    # Install GVM-Tools
-    #
-    
-RUN pip3 install gvm-tools==$gvm_tools_version && \
-    echo "/usr/local/lib" > /etc/ld.so.conf.d/openvas.conf && ldconfig && cd / && rm -rf /build
+RUN { \
+    echo '@custcom /repo/community/'; \
+    echo 'https://dl-5.alpinelinux.org/alpine/v3.14/main/' ; \
+    echo 'https://dl-5.alpinelinux.org/alpine/v3.14/community/' ;\
+    echo 'https://dl-4.alpinelinux.org/alpine/v3.14/main/' ; \
+    echo 'https://dl-4.alpinelinux.org/alpine/v3.14/community/' ;\
+    echo 'https://dl-cdn.alpinelinux.org/alpine/v3.14/main/' ; \
+    echo 'https://dl-cdn.alpinelinux.org/alpine/v3.14/community/' ; \
+    } >/etc/apk/repositories \
+    && cat /etc/apk/repositories \
+    && sleep 5 \
+    && apk update --update-cache \
+    && sleep 5 \
+    && apk upgrade --available \
+    && sleep 5 \
```

Comment:
These sleep commands are not required, bash writes to files synchroneously, and apk blocks until it's complete.

As specified in [the Alpine wiki](https://wiki.alpinelinux.org/wiki/Edge), these two apk commands can be folded into a single command.

Line 53 to Line 58 should be
```
&& apk update --update-cache --available \
```

Reply:
The sleep is there because sometimes, it would else break the build.

It can be shorten thats correct.